src/igvfd/audit/measurement_set.py

```diff
@@ -134,25 +135,22 @@ def audit_inconsistent_modifications(value, system):
         yield AuditFailure('inconsistent modifications', detail, level='NOT_COMPLIANT')
 
 
-@audit_checker('MeasurementSet', frame='object')
+@audit_checker('MeasurementSet', frame='embedded')
```

Comment:
we should not rely on embedding in my opinion

Reply:
reverted to object